app/notification.go

```diff
@@ -255,6 +281,23 @@ func (a *App) SendNotifications(post *model.Post, team *model.Team, channel *mod
 		updateMentionChans = append(updateMentionChans, umc)
 	}
 
+	notifyCRT := &CRTNotifiers{}
```

Comment:
nit: `notifyCRT` seems a little ambiguous to me. What about something like `notificationsForCRT`?